icekit/project/settings/_base.py

```diff
@@ -414,18 +414,18 @@
 # given model/field, without generating additional unecessary thumbnails. This
 # is essential when using a remote storage backend.
 THUMBNAIL_ALIASES = {
-#     'app[.model][.field]': {
-#         'name-WxH': {
-#             'size': (W, H),
-#             ...,
-#         },
-#     },
+    # 'app[.model][.field]': {
+    #   'name-WxH': { 'size': (W, H), },
+    # },
     '': {
+        'author_portrait': {
+            'size': (360, 640),
+        },
```

Comment:
Fair enough, if we want to use the alias in templates that exist outside the authors app, then we need a global alias. And you're probably right that authors_portrait probably won't clash (or the wrong size). But it still feels like chance rather than good practice.

Previously (SFMOMA) I believe we didn't use aliases at all, which is probably what made it hard to find and replace all occurrences when designs changed. We only started using aliases when we started resolving scaling issues (offline compression, generating thumbnails at upload time, etc.)

So just by using any alias at all (even one that includes WxD), I think we are already much better off in terms of updating templates if/when the dimensions change.

I don't think it's a bad idea to include the WxD in the alias name specifically because:

1. we used to embed WxD in the template anyway, before we used aliases;
2. it avoids the issue where any other template (from another app or our own apps) needs to know beforehand the dimensions of the alias and that it is correct for the template;
3. it's probably easier to find and replace `authors_portrait_WxD` than just `authors_portrait`;
4. it's one clear convention to follow -- we don't find ourselves in a situation where actually someone does want to display a differently sized version of a portrait and then has to decide on a name like `authors_portrait_small` (and then if there's another template that needs `authors_portrait_xsmall`, and then maybe one of the templates changes and small is now smaller than xsmall).

I think it's clearer to include WxD to disambiguate the size requirement than words that describe the template/name/purpose/size. The name component of the alias should still be as descriptive as possible to avoid conflicts, but it should define the image source (not where it is used). We shouldn't care where/how/why it is used. Just what it is and what size it is.

Also, note that with offline generation of thumbnails, global thumbnails will be generated for *all global aliases* for all images that are uploaded. So we will have image fields on models completely unrelated to author portraits, having an "authors_portrait" thumbnail generated. The app/model namespace helps us only generate those thumbnails we need for the authors portrait image when it is uploaded.